app/javascript/tag-edit-button.vue

```diff
@@ -43,7 +43,13 @@ export default {
     updateTag(name) {
       this.tagName = name
       this.toast('タグ名を変更しました')
-      location.href = `/questions/tags/${encodeURIComponent(name)}?all=true`
+      const url = location.pathname.split('/')
+      const path = url[url.length - 3]
+      if (path === 'questions') {
+        location.href = `/questions/tags/${encodeURIComponent(name)}?all=true`
+      } else if (path === 'pages') {
+        location.href = `/pages/tags/${encodeURIComponent(name)}?all=true`
+      }
```

Comment:
URLの差異が分かりやすくなるように、三項演算子にするのはどうでしょう？

```suggestion
      const tagType = path === 'questions' ? 'questions' : 'pages'
      location.href = `/${tagType}/tags/${encodeURIComponent(name)}?all=true`
```

Reply:
@sinsoku さん
レビュー頂きありがとうございます!🙏

`tagType`という変数を新たに設定し、If・elseの部分を三項演算子する方がコードも短くなり分かりやすいですね！
大変勉強になりました！ありがとうございます😀😀